sailpoint.sgnl.yaml

```diff
@@ -0,0 +1,1427 @@
+description: "Sailpoint as a System of Record"
+address: "{{Input Required: Sailpoint URL}}"
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "SCIM2.0-1.0.0"
+# Example Configs:
+# empty config: {} | b64: "e30="
+# config with filtering on Users: {"Users":{"queryParams":{"filter":"displayName eq \"SGNL\""}}} | b64: "eyJVc2VycyI6eyJxdWVyeVBhcmFtcyI6eyJmaWx0ZXIiOiJkaXNwbGF5TmFtZSBlcSBcIlNHTkxcIiJ9fQ=="
+# config with filter, ordering and sortBy on Groups {"Groups":{"queryParams":{"filter":"displayName eq \"SGNL\"","sortBy":"userName","ascending":true}}} | b64: "eyJHcm91cHMiOnsicXVlcnlQYXJhbXMiOnsiZmlsdGVyIjoiZGlzcGxheU5hbWUgZXEgXCJTR05MXCIiLCJzb3J0QnkiOiJ1c2VyTmFtZSIsImFzY2VuZGluZyI6dHJ1ZX19fQ=="
+adapterConfig: "e30=" # {}
+# All auth mechanisms specified below must be supported by the specified Adapter.
+auth:
+  - basic:
+      username: "{{Input Required: Username}}"
+      password: "{{Input Required: API Token}}"
+entities:
+  Account:
+    displayName: SailpointAccount
+    externalId: Accounts
+    description: Entity representing a Sailpoint Identity IQ Account.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: nativeIdentity
+        externalId: nativeIdentity
+        type: String
+      - name: displayName
+        externalId: displayName
+        type: String
+      - name: instance
+        externalId: instance
+        type: String
+      - name: uuid
+        externalId: uuid
+        type: String
+      - name: active
+        externalId: active
+        type: Bool
+      - name: locked
+        externalId: locked
+        type: Bool
+      - name: lastRefresh
+        externalId: lastRefresh
+        type: DateTime
+      - name: lastTargetAggregation
+        externalId: lastTargetAggregation
+        type: DateTime
+      - name: manuallyCorrelated
+        externalId: manuallyCorrelated
+        type: Bool
+      - name: hasEntitlements
+        externalId: hasEntitlements
+        type: Bool
+      - name: identityValue
+        externalId: $.identity.value
+        type: String
+      - name: identityRef
+        externalId: $.identity.ref
+        type: String
+      - name: identityUserName
+        externalId: $.identity.userName
+        type: String
```

Comment:
We might want to index this.
We don't index this in other SoR templates (e.g. SNOW), so I'm ok with leaving this unindexed for now.

Reply:
Indexed the attributes suggested in the comments.